packages/falcon-front-kit/src/DynamicRoute/DynamicRoute.tsx

```
@@ -43,7 +40,9 @@ export const DynamicRoute: React.SFC<DynamicRouteProps> = props => {
 
               const Component = components[url.type];
               if (!Component) {
-                return <Text>{`Please register component for '${url.type}' content type!`}</Text>;
```

@tutok I would probably keep the message in the body (at least in `development` mode). We can only hope a developer would keep the console tab open during the development, but we cannot guarantee that hence it will confuse new developers with what to do next and why their app does not show anything.

What do you think guys? or maybe `return <NotFound location={location} />;` on line 45?

cc: @mateuszwozniak

---

It might be a good idea for the development mode indeed (just print a warning like it was before).  I don't want to return `NotFound` as that would be misleading - the content have been found but the server doesn't know how to render it. Returning `NotFound` would indicate that passed url cannot be resolved (nobody is able to handle the url) and that's not true in that case.

---

@mateuszwozniak so maybe throw an Error and render a nice "error page" in dev mode? (https://github.com/deity-io/falcon/issues/262)

---

Yes, that would be much better.

---

done